src/modules/count.js

```diff
@@ -0,0 +1,13 @@
+const commentsCount = () => {
+  const comments = document.querySelectorAll('.comments');
+  return comments.length;
+};
+
+const itemsCount = async () => {
+  const sourceURL = 'https://api.tvmaze.com/seasons/1/episodes';
```

Comment:
- [ ] You have done well by declaring the` source url` as a variable that can be accessible anywhere in your project, kindly avoid code repetition as we have it in **displayList.js file line 6**. Doing this will ensure that you follow DRY principle. 👍